internal/broadcast/definition.go

```diff
@@ -24,21 +24,65 @@ import (
 	"github.com/hyperledger-labs/firefly/pkg/fftypes"
 )
 
-func (bm *broadcastManager) broadcastDefinitionAsNode(ctx context.Context, def fftypes.Definition, tag fftypes.SystemTag, waitConfirm bool) (msg *fftypes.Message, err error) {
-	signingIdentity, err := bm.GetNodeSigningIdentity(ctx)
+func (bm *broadcastManager) BroadcastDefinitionAsNode(ctx context.Context, def fftypes.Definition, tag fftypes.SystemTag, waitConfirm bool) (msg *fftypes.Message, err error) {
+	return bm.BroadcastDefinition(ctx, def, &fftypes.Identity{ /* resolve to node default */ }, tag, waitConfirm)
+}
+
+// THIS IS WHERE THE OLD CODE FIRST CALLS ResolveInputIdentity
+// It does NOT have an author and key the first time
```

Comment:
This comment feels like it can be removed. Don't think there's any need to reference OLD CODE vs NEW CODE, but we should note if there's a specific contract coming into a method (ie the identity must/must not be resolved when calling this).